doc/admin-guide/configuration/hrw4u.en.rst

```diff
@@ -20,9 +20,9 @@
 HRW4U DSL
 *********
 
-This is a next-generation rule language for the existing ATS ``header_rewrite`` plugin. It
-introduces a clearer syntax, formal grammar, full condition nesting, semantic validation,
-and extensible condition/operator support.
+HRW4U is a next-generation rule language for the existing ATS ``header_rewrite``
```

Comment:
I see we link to it a few lines below, but, since this is the first instance, perhaps, 
```
:ref:`admin-plugins-header-rewrite`
```